conf/modules.config

```
@@ -17,173 +17,91 @@ process {
         publishDir = [
-            path: { "$params.outdir/genome_properties/katgc" },
+            path: { "${params.outdir}/QC/${task.process.tokenize(':')[-3].toLowerCase()}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
             mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }            
         ]
     }
-
-    // COMPARE_LIBRARIES
-    // withName: 'MERQURYFK_KATCOMP' {
-    //     publishDir = [
-    //         path: { "$params.outdir/library_comparison/kat_comp" },
-    //         mode: params.publish_dir_mode,
-    //         saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-    //     ]
-    // }
-
-    // CONTAMINATION SCREEN
-    withName: 'MASH_SCREEN' {
-        ext.args   = '-w'
-        ext.prefix = { "${meta.id}_${query.baseName}" }
+    // QUAST: Prefer to keep the local module since it can deal with the inputs I have
+    withName: QUAST {
+        ext.prefix = '"${meta.id}"-"${task.process}.tokenize(":")[-3].toLowerCase()"'
         publishDir = [
-            path: { "$params.outdir/contamination_screen/mash_screen/screens" },
+            path: { "${params.outdir}/QC/${task.process.tokenize(':')[-3].toLowerCase()}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
             mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }            
         ]
     }
-    withName: 'MASH_FILTER' {
-        // Filter hits with greater than 90% identity and greater than 100 shared hashes
-        ext.args   = "-F '[\\t/]' '\$1 > 0.9 && \$2 > 100'"
+    // MERQURY: use nf-core module
+    withName: MERQURY {
+        ext.prefix = '"${meta.id}"-"${task.process}.tokenize(":")[-3].toLowerCase()"'
         publishDir = [
-            path: { "$params.outdir/contamination_screen/mash_screen" },
+            path: { "${params.outdir}/QC/${task.process.tokenize(':')[-3].toLowerCase()}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
             mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }            
         ]
     }
-
-    // COMPARE_ASSEMBLIES
-    withName: 'QUAST' {
-        cpus       = { Math.min(6, consensus instanceof List ? consensus.size() : 1 ) }
+    /* 
+    ----------
+    Polishing
+    ----------
+    */
+    // Medaka: switch to nf-core
+    withName: MEDAKA {
+        ext.args = [ params.medaka_model ? "--model ${params.medaka_model}" : '' ]
+        ext.prefix = '"${meta.id}_medaka"'
```

I hdont recall the use of ' around args, why is that used here? I'd rather propose
```suggestion
        ext.prefix = { "${meta.id}_medaka" }
```